makiflow/layers/untrainable_layers.py

```diff
@@ -195,22 +210,24 @@ def __call__(self, x: list):
         )
         return maki_tensor
 
-    def _forward(self, X):
-        return sum(X)
+    def _forward(self, X, type_graph_operation=MakiRestorable.TEST_PREFIX):
+        with tf.name_scope(type_graph_operation + super().get_name()):
+            return sum(X)
```

Comment:
Add a commentary on why there is `sum` instead of `tf.reduce_sum` or `tf.add_n`.